client/src/components/pages/RobotInteractionCockpit/RobotInteractionCockpit.jsx

```diff
@@ -120,6 +200,55 @@ const RobotInteractionCockpit = (match) => {
                 </Button>
               </>
             )}
+          {currentStep !== 0 && (
+            <>
+              <Row>
+                <Col span={12}>
+                  <Title
+                    style={{
+                      marginBottom: '0px',
+                      marginLeft: '10px',
+                    }}
+                    level={5}
+                  >
+                    Robot Status
+                  </Title>
+                  <Card style={{ margin: '10px' }} hoverable>
+                    <Row>
+                      <Col span={18}>
+                        <Title
+                          style={{
+                            marginBottom: '0px',
+                            top: '35%',
+                            position: 'absolute',
+                          }}
+                          level={5}
+                        >
+                          {robotState.toLocaleUpperCase()}
+                        </Title>
+                      </Col>
+                      <Col span={6}>{displayStatusIcon(robotState)}</Col>
+                    </Row>
+                  </Card>
+                </Col>
+                <Col span={12}>
```

Comment:
I would prefer the column widths to be dynamically adjusted. On smaller devices, the left column should be significantly smaller than the right column. Otherwise I don't think it's so easy to read.

Please check this for all used columns and provide a video for responsiveness (or short video call).